<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtuexportwidgetClass</class>
 <widget class="QWidget" name="dtuexportwidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导出</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="Text_1">
     <property name="text">
      <string>导出内容:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lab_what">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>XX报告</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="Text_3">
     <property name="text">
      <string>当前总数:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lab_count">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Text_2">
     <property name="text">
      <string>导出范围:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spin_min">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>--&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spin_max">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btn_export">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_export</sender>
   <signal>clicked()</signal>
   <receiver>dtuexportwidgetClass</receiver>
   <slot>export_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>export_all()</slot>
 </slots>
</ui>
